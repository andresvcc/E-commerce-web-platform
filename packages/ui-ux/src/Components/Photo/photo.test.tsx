import React from 'react';
import { describe, expect, test } from '@jest/globals';
import renderer from 'react-test-renderer';
import { Photo } from './index';

describe('tests Photo component', () => {
  test('test Photo rendering', () => {
    const tree = renderer.create(<Photo src="logo-mobsya" />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
