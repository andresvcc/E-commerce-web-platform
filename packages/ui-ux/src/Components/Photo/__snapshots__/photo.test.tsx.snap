// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests Photo component test Photo rendering 1`] = `
<div
  className="sc-gEvEer puYPP"
>
  <img
    className="sc-aXZVg cwsYjk"
    src={{}}
  />
  <h5>
    logo-mobsya
  </h5>
</div>
`;
